src/Dockerfile

```diff
@@ -9,11 +9,17 @@ ENV ASPNETCORE_HTTP_PORTS=8080
 
 WORKDIR /app
 COPY services/EdFi.DataManagementService.Api/*.csproj ./services/EdFi.DataManagementService.Api/
+COPY backend/EdFi.DataManagementService.Backend/*.csproj ./backend/EdFi.DataManagementService.Backend/
+COPY backend/EdFi.DataManagementService.Backend.Mssql/*.csproj ./backend/EdFi.DataManagementService.Backend.Mssql/
+COPY backend/EdFi.DataManagementService.Backend.Postgresql/*.csproj ./backend/EdFi.DataManagementService.Backend.Postgresql/
 COPY EdFi.DataManagementService-Docker.sln ./
 COPY nuget.config .
 RUN dotnet restore
 
 COPY services/EdFi.DataManagementService.Api ./services/EdFi.DataManagementService.Api
+COPY backend/EdFi.DataManagementService.Backend ./backend/EdFi.DataManagementService.Backend
+COPY backend/EdFi.DataManagementService.Backend.Mssql ./backend/EdFi.DataManagementService.Backend.Mssql
+COPY backend/EdFi.DataManagementService.Backend.Postgresql ./backend/EdFi.DataManagementService.Backend.Postgresql
```

Comment:
For a long time I forgot about the [.dockerignore](https://docs.docker.com/build/building/context/#dockerignore-files). I wonder if that would be useful for simplifying the `Dockerfile`?

Reply:
Good call. Done.